tests/parm/ccpp_regional.nml.IN

```diff
@@ -89,17 +89,18 @@
        hord_vt = -5
        hord_tm = -5
        hord_dp = -5
-       hord_tr =  8
+       hord_tr = -8
```

Comment:
@XiaqiongZhou-NOAA would you please review this, I thought you updated the code so that all the hord parameters to be positive? Maybe I missed something.

Reply:
hord_tr should be positive 8. No negative value except -5.

Reply:
New values:
       hord_mt =  6
       hord_vt =  6
       hord_tm =  6
       hord_dp =  6
       hord_tr =  10